libcudacxx/test/libcudacxx/std/utilities/template.bitset/bitset.members/op_and_eq.pass.cpp

```diff
@@ -52,8 +52,7 @@ int main(int, char**)
   test_op_and_eq<64>();
   test_op_and_eq<65>();
   test_op_and_eq<1000>(); // not in constexpr because of constexpr evaluation step limits
-// 11.4 added support for constexpr device vars needed here
-#if TEST_STD_VER >= 2014 && _CCCL_CUDACC_AT_LEAST(11, 4)
+  // 11.4 added support for constexpr device vars needed here
```

Comment:
Remark: maybe we could remove such anecdotes from history. Not sure. @miscco?

Reply:
yeah that would be fine